<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="ValidationException.xml" source-language="en-US" target-language="pt-BR">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5e7f076d39684afc47faa3f7aac66f5782924a98a.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">e7f076d39684afc47faa3f7aac66f5782924a98a</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-1.0,netcore-1.1,netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>Represents the exception that occurs during validation of a data field when the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /&gt;</ph> class is used.</source>
          <target state="translated">Representa a exceção que ocorre durante a validação de um campo de dados quando a classe <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /&gt;</ph> é usada.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>A validation exception occurs if an input value does not match the expected data type, range or pattern of the data field.</source>
          <target state="translated">Uma exceção de validação ocorre se um valor de entrada não coincide com o tipo de dados esperado, o intervalo ou o padrão do campo de dados.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>For example, if a user enters an integer value in a data field that expects a <ph id="ph1">&lt;xref:System.Data.SqlDbType.DateTime&gt;</ph> value, a validation exception occurs.</source>
          <target state="translated">Por exemplo, se um usuário insere um valor inteiro em um campo de dados que espera um <ph id="ph1">&lt;xref:System.Data.SqlDbType.DateTime&gt;</ph> valor, ocorrerá uma exceção de validação.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>This class uses the <ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationAttribute&gt;</ph> class to customize validations.</source>
          <target state="translated">Essa classe usa a <ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationAttribute&gt;</ph> classe para personalizar as validações.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>A <ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationException&gt;</ph> is thrown if a validation error occurs.</source>
          <target state="translated">Um <ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationException&gt;</ph> é lançada se ocorrer um erro de validação.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>The exception is thrown when the <ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationAttribute.Validate%2A&gt;</ph> method is called.</source>
          <target state="translated">A exceção é gerada quando o <ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationAttribute.Validate%2A&gt;</ph> método é chamado.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>All the exceptions that are thrown during validation are contained in the <ph id="ph1">&lt;xref:System.Web.DynamicData.IDynamicValidatorException.InnerExceptions%2A&gt;</ph> collection.</source>
          <target state="translated">Todas as exceções geradas durante a validação estão contidas no <ph id="ph1">&lt;xref:System.Web.DynamicData.IDynamicValidatorException.InnerExceptions%2A&gt;</ph> coleção.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>You can retrieve each validation exception by iterating through the <ph id="ph1">&lt;xref:System.Web.DynamicData.IDynamicValidatorException.InnerExceptions%2A&gt;</ph> collection.</source>
          <target state="translated">Você pode recuperar cada exceção de validação pela iteração a <ph id="ph1">&lt;xref:System.Web.DynamicData.IDynamicValidatorException.InnerExceptions%2A&gt;</ph> coleção.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> class.</source>
          <target state="translated">Inicializa uma nova instância da classe <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> class using an error message generated by the system.</source>
          <target state="translated">Inicializa uma nova instância da classe <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> usando uma mensagem de erro gerada pelo sistema.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor">
          <source>This method is the default constructor for the <ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationException&gt;</ph> class.</source>
          <target state="translated">Esse método é o construtor padrão para o <ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationException&gt;</ph> classe.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String)">
          <source>A specified message that states the error.</source>
          <target state="translated">Uma mensagem especificada que indica o erro.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> class using a specified error message.</source>
          <target state="translated">Inicializa uma nova instância da classe <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> usando uma mensagem de erro especificada.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String)">
          <source>For more information about how to use this constructor, see <bpt id="p1">[</bpt>How to: Customize Data Field Validation in the Data Model<ept id="p1">](http://msdn.microsoft.com/library/47d3b04e-1ab5-466d-bc58-92f163f3d7f5)</ept>.</source>
          <target state="translated">Para obter mais informações sobre como usar esse construtor, consulte <bpt id="p1">[</bpt>como: personalizar validação de campo de dados no modelo de dados<ept id="p1">](http://msdn.microsoft.com/library/47d3b04e-1ab5-466d-bc58-92f163f3d7f5)</ept>.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The object that holds the serialized data.</source>
          <target state="translated">O objeto que contém os dados serializados.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Context information about the source or destination of the serialized object.</source>
          <target state="translated">As informações de contexto sobre a origem ou o destino do objeto serializado.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> class using serialized data.</source>
          <target state="translated">Inicializa uma nova instância da classe <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> usando dados serializados.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This constructor is called during deserialization to reconstitute the exception object that is transmitted over a stream.</source>
          <target state="translated">Este construtor é chamado durante a desserialização para reconstituir o objeto de exceção que é transmitido em um fluxo.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.Exception)">
          <source>The error message.</source>
          <target state="translated">A mensagem de erro.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.Exception)">
          <source>The collection of validation exceptions.</source>
          <target state="translated">A coleção de exceções de validação.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> class using a specified error message and a collection of inner exception instances.</source>
          <target state="translated">Inicializa uma nova instância da classe <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> usando uma mensagem de erro e uma coleção de instâncias de exceção interna especificadas.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.Exception)">
          <source>You typically will not use this constructor because validation exceptions are stored in the <ph id="ph1">&lt;xref:System.Web.DynamicData.IDynamicValidatorException.InnerExceptions%2A&gt;</ph> property, not <ph id="ph2">`innerException`</ph>.</source>
          <target state="translated">Você normalmente não usará este construtor como exceções de validação são armazenadas no <ph id="ph1">&lt;xref:System.Web.DynamicData.IDynamicValidatorException.InnerExceptions%2A&gt;</ph> propriedade, não <ph id="ph2">`innerException`</ph>.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.ComponentModel.DataAnnotations.ValidationResult,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>The list of validation results.</source>
          <target state="translated">A lista de resultados de validação.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.ComponentModel.DataAnnotations.ValidationResult,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>The attribute that caused the current exception.</source>
          <target state="translated">O atributo que causou a exceção atual.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.ComponentModel.DataAnnotations.ValidationResult,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>The value of the object that caused the attribute to trigger the validation error.</source>
          <target state="translated">O valor do objeto que fez com que o atributo disparasse o erro de validação.</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.ComponentModel.DataAnnotations.ValidationResult,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> class by using a validation result, a validation attribute, and the value of the current exception.</source>
          <target state="translated">Inicializa uma nova instância da classe <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> usando um resultado de validação, um atributo de validação e o valor da exceção atual.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>The message that states the error.</source>
          <target state="translated">A mensagem que indica o erro.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>The attribute that caused the current exception.</source>
          <target state="translated">O atributo que causou a exceção atual.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>The value of the object that caused the attribute to trigger validation error.</source>
          <target state="translated">O valor do objeto que fez com que o atributo disparasse o erro de validação.</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> class using a specified error message, a validation attribute, and the value of the current exception.</source>
          <target state="translated">Inicializa uma nova instância da classe <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> usando a mensagem de erro, o atributo de validação e o valor da exceção atual especificados.</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>For more information, see How to: Customize Data Field Validation in the Data Model.</source>
          <target state="translated">Para obter mais informações, consulte como: personalizar validação de campo de dados no modelo de dados.</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" uid="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationAttribute">
          <source>Gets the instance of the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /&gt;</ph> class that triggered this exception.</source>
          <target state="translated">Obtém a instância da classe <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /&gt;</ph> que disparou essa exceção.</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationAttribute">
          <source>An instance of the validation attribute type that triggered this exception.</source>
          <target state="translated">Uma instância do tipo de atributo de validação que disparou a essa exceção.</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationAttribute">
          <source>This type is used to enforce validation based on the metadata associated with the data table.</source>
          <target state="translated">Esse tipo é usado para impor a validação com base nos metadados associados à tabela de dados.</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationAttribute">
          <source>If an error occurs, the <ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationAttribute&gt;</ph> class triggers a <ph id="ph2">&lt;xref:System.ComponentModel.DataAnnotations.ValidationException&gt;</ph>.</source>
          <target state="translated">Se ocorrer um erro, o <ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationAttribute&gt;</ph> classe gatilhos um <ph id="ph2">&lt;xref:System.ComponentModel.DataAnnotations.ValidationException&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="136" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationAttribute">
          <source>This class can be overridden to create custom validation attributes.</source>
          <target state="translated">Essa classe pode ser substituída para criar atributos de validação personalizada.</target>       </trans-unit>
        <trans-unit id="137" translate="yes" xml:space="preserve" uid="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationResult">
          <source>Gets the <ph id="ph1">&lt;see cref="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationResult" /&gt;</ph> instance that describes the validation error.</source>
          <target state="translated">Obtém a instância <ph id="ph1">&lt;see cref="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationResult" /&gt;</ph> que descreve o erro de validação.</target>       </trans-unit>
        <trans-unit id="138" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationResult">
          <source>The <ph id="ph1">&lt;see cref="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationResult" /&gt;</ph> instance that describes the validation error.</source>
          <target state="translated">O <ph id="ph1">&lt;see cref="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationResult" /&gt;</ph> a instância que descreve o erro de validação.</target>       </trans-unit>
        <trans-unit id="139" translate="yes" xml:space="preserve" uid="P:System.ComponentModel.DataAnnotations.ValidationException.Value">
          <source>Gets the value of the object that causes the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /&gt;</ph> class to trigger this exception.</source>
          <target state="translated">Obtém o valor do objeto que faz com que a classe <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /&gt;</ph> dispare essa exceção.</target>       </trans-unit>
        <trans-unit id="140" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.ComponentModel.DataAnnotations.ValidationException.Value">
          <source>The value of the object that caused the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /&gt;</ph> class to trigger the validation error.</source>
          <target state="translated">O valor do objeto que causou a <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /&gt;</ph> classe para disparar o erro de validação.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>